frame/staking/src/tests.rs

```diff
@@ -513,8 +514,8 @@ fn nominating_and_rewards_should_work() {
 					total: 1000 + 800,
 					own: 1000,
 					others: vec![
-						IndividualExposure { who: 3, value: 400 },
 						IndividualExposure { who: 1, value: 400 },
+						IndividualExposure { who: 3, value: 400 },
```

Comment:
these are because we're altering the iteration order of snapshot here.

Reply:
Yeah, I figured.